com.unity.perception/Runtime/GroundTruth/Labelers/BoundingBoxLabeler.cs

```diff
@@ -108,8 +116,73 @@ void OnRenderedObjectInfosCalculated(int frameCount, NativeArray<RenderedObjectI
                     };
                 }
 
+                if (!CaptureOptions.useAsyncReadbackIfSupported && frameCount != Time.frameCount) 
```

Comment:
Looks like the current boxes are not in sync with the frame. I guess it's because we aren't reading back synchronously while the UI is up?

Reply:
So I put a warning logging statement in if we are rendering results from a different frame than the one that we are currently on, and then doesn't log, so I don't know why. I think (although I'm not 100% sure anymore) that I was more in sync with Priyesh's fixes, and being a frame off is definitely something that he had mentioned. So I was putting all of my eggs in the basket of his fix fixing it. If not then some fun debugging.

Reply:
I don't think so. Can you also repro this?

Reply:
Fixed (as you know since you helped)